setup_hooks.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
+chmod 0755 $PWD/scripts/pre-push
```

Comment:
can we just do `chmod +x` instead of setting all of the permissions?

Reply:
Sure...